### ISSUE TITLE
Error on OCLint 0.9 dev build downloads

### ISSUE THREAD

**Issue by wooster** [oclint/oclint#189]
The [downloads](http://oclint.org/downloads.html) for the OCLint 0.9 dev builds return an error:

``` xml
<Error>
<Code>InvalidObjectState</Code>
<Message>
The operation is not valid for the object's storage class
</Message>
<RequestId>1D2114A6B89A0CBC</RequestId>
<HostId>
Abnena56BwOU7cwI+jBcH5SLMpIGZPrtImmFiQ9vx+6WNVNuSI/xl1ddH0DJMsoX
</HostId>
</Error>
```

**Comment by garrettmoon** [comment]
They're invalid again.

**Comment by ryuichisaito6** [comment]
@garrettmoon oops, time flies. ;) Will generate one pretty soon.

**Comment by ryuichisaito6** [comment]
@garrettmoon updated the binary with some changes for Xcode 7.

**Comment by xfreebird** [comment]
:+1:

**Comment by isnotnull** [comment]
Unavailable again.

**Comment by ryuichis** [comment]
@isnotnull please go to https://github.com/oclint/oclint/releases for the latest release.

**Comment by isnotnull** [comment]
@ryuichis Thanks.
